apps/server/src/modules/files-storage-client/mapper/error.mapper.spec.ts

```diff
@@ -3,90 +3,49 @@ import {
 	ConflictException,
 	ForbiddenException,
 	InternalServerErrorException,
-	ValidationError as IValidationError,
 } from '@nestjs/common';
-import { ApiValidationError } from '@shared/common';
 import _ from 'lodash';
-import { IFileStorageErrors } from '../interfaces';
+import { IError } from '@shared/infra/rabbitmq';
 import { ErrorMapper } from './error.mapper';
 
 describe('ErrorMapper', () => {
-	const createApiValidationError = (): ApiValidationError => {
-		const constraints: IValidationError[] = [];
-		constraints.push(
-			{
-				property: 'propWithoutConstraint',
-			},
-			{
-				property: 'propWithOneConstraing',
-				constraints: {
-					rulename: 'ruleDescription',
-				},
-			},
-			{
-				property: 'propWithMultipleCOnstraints',
-				constraints: {
-					rulename: 'ruleDescription',
-					secondrulename: 'secondRuleDescription',
-				},
-			}
-		);
+	describe('mapErrorToDomainError', () => {
+		it('Should map any 400 error to BadRequestException.', () => {
+			const errorText = 'BadRequestException ABC';
+			const e = new BadRequestException(errorText);
+			const json = _.toPlainObject(e) as IError;
```

Comment:
IMO it is not correct to call this "json" because JSON is a string and this is an object.

It also seems a bit complicated to do it this way. Why not?
```suggestion
			const error = { name: 'error name', message: errorText, status: 400} as IError;
```

Reply:
Das ist ein Object in json Format. Sicher könnte man die const anders benennen, aber ein Mehrwert bringt das nicht.